src/atoms/Text/Text.native.js

```diff
@@ -0,0 +1,87 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import styled from 'styled-components/native';
```

Comment:
```suggestion
import styled from 'styled-components';
```

Reply:
We need /native

Reply:
This doesn't work in react native. There's an alternative to make it work [here](https://styled-components.com/docs/basics#simpler-usage-with-the-metro-bundler). But that's again an issue with storybook and the consumers will also need to make their metro work like this.